GoogleForCreators/web-stories-wp
# Editor: Empty state

## Summary

## Relevant Technical Choices



## To-do


## User-facing changes

Empty story (with no elements added and default background changed) will now display this message until you make it non-empty:
![image](https://user-images.githubusercontent.com/21036224/150830442-f74845cb-c3cf-4d56-b49d-715ad4871942.png)


## Testing Instructions






This PR can be tested by following these steps:

1. Create a new story - it should display an empty story message
2. Add something (text, image, shape etc.) - the message should dissapear
3. Verify that all actions from the context menu are working


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #10110
